browser-interface/scripts/copyBuiltFiles.ts

```diff
@@ -31,6 +32,12 @@ async function copyBuiltFiles() {
     copyFile(file, path.resolve(DIST_PATH, file.replace(basePath + '/', './')))
   }
 
+  const webGLStreamingAssetsPath = path.resolve(process.env.BUILD_PATH!, 'StreamingAssets/aa/WebGL')
```

Comment:
why is it /aa/?

Reply:
Its the default `LocalBuild` path by Unity. We could remove the aa if we wanted to
<img width="830" alt="image" src="https://user-images.githubusercontent.com/1999557/223757938-e302cd29-382c-49f8-97c0-25eee5f5cf55.png">